ui/client/components/toolbars/status/ProcessInfo.tsx

```diff
@@ -111,10 +124,7 @@ class ProcessInfo extends React.Component<OwnProps & StateProps, State> {
             </CssFade>
           </SwitchTransition>
           <ToolbarButtons>
-            <SaveButton/>
-            <Deploy/>
-            <Cancel/>
-            <Metrics/>
+            {[...this.buttons, ...customButtons]}
```

Comment:
Let's add some comment/TODO to design it better for the case where there are multiple buttons?

Reply:
:+1: